application/extensions/admin/grid/MassiveActionsWidget/assets/listActions.js

```diff
@@ -369,28 +369,28 @@ $(document).off('pjax:scriptcomplete.listActions').on('pjax:scriptcomplete.listA
 
 
 function switchStatusOfListActions(e) {
-    var checkboxes = $('input[type="checkbox"]');
-    var actionButton = $('.massiveAction');
+    var checkboxSelector = $('#usermanagement--action-toggleAllUsers, .usermanagement--selector-userCheckbox');
```

Comment:
Maybe a better selector ? We don't use it on other Batch actions ?

`input[type="checkbox"]` is clearly too large …